src/game/Entities/Unit.cpp

```diff
@@ -4647,7 +4647,7 @@ void Unit::SetFacingTo(float ori)
     init.Launch();
     // orientation change is in-place
     UpdateSplinePosition();
-    movespline->_Finalize();
+    DisableSpline();
```

Comment:
Disable calls _Interrupt, I need to call _Finalize. The way this works is that on blizz end, orientation SMSG_MONSTER_MOVE is immediate and is the reason why its this way. I understand that prolly the "MOVEFLAG_SPLINE_ENABLED | MOVEFLAG_FORWARD" is required, so a new function is in order called EndSpline. Please test if a new func with _Finalize and the moveflag change does the trick.

Reply:
@killerwife  so, like: https://github.com/cmangos/mangos-tbc/commit/cc5eacc000d3ab9bc0d9235ab4a5cb804f683c9a ?

I haven't yet been able to test this on my live server, but it's the gist, right?

Reply:
aye, does do the trick ☝️

Reply:
Amazing.

Reply:
Add the commit to the PR then.

Reply:
I have no edit permissions on this PR 😞 

Opened a new one: https://github.com/cmangos/mangos-tbc/pull/647